IDEAS/Buildings/Components/Window.mo

```diff
@@ -45,6 +53,13 @@ model Window "Multipane window"
         extent={{10,-10},{-10,10}},
         rotation=-90,
         origin={-40,-100})));
+
+  Modelica.Thermal.HeatTransfer.Components.ThermalConductor theBri(final G=briType.G) if
+       briType.present "Themal bridge of the window perimeter"
```

Comment:
That would work for a wall (which also includs thermall mass of a bridge), but very complex for a window.